apps/src/templates/studioHomepages/SetUpMessage.jsx

```diff
@@ -46,7 +48,7 @@ class SetUpMessage extends Component {
           href={buttonUrl}
           onClick={onClick}
           className={buttonClass}
-          color={Button.ButtonColor.gray}
+          color={buttonColor || Button.ButtonColor.gray}
```

Comment:
nit: it's more idiomatic to set a default for this prop rather than having gray be the fallback option. example:

```javascript
function MyComponent({buttonColor}) {
  return <button color={buttonColor}>hi</button>
}

MyComponent.defaultProps = {
  buttonColor: Button.ButtonColor.gray
}
```

Reply:
Good call––just updated it. I'm thinking when I refactor this, will change it to a function component